eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -92,6 +91,13 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.executeUpdate("CREATE INDEX metrics_name_idx ON audit.path_finding_metrics(experiment_name)")
       }
 
+      def migration910(statement: Statement): Unit = {
+        statement.executeUpdate("CREATE TABLE audit.transaction_published (tx_id TEXT NOT NULL PRIMARY KEY, channel_id TEXT NOT NULL, node_id TEXT NOT NULL, fee_sat BIGINT NOT NULL, tx_type TEXT NOT NULL, timestamp TIMESTAMP WITH TIME ZONE NOT NULL)")
+        statement.executeUpdate("CREATE TABLE audit.transaction_confirmed (tx_id TEXT NOT NULL PRIMARY KEY, channel_id TEXT NOT NULL, node_id TEXT NOT NULL, timestamp TIMESTAMP WITH TIME ZONE NOT NULL)")
+        statement.executeUpdate("CREATE INDEX transaction_published_timestamp_idx ON audit.transaction_published(timestamp)")
+        statement.executeUpdate("CREATE INDEX transaction_confirmed_timestamp_idx ON audit.transaction_confirmed(timestamp)")
+      }
```

Comment:
Having the old table and the new ones together makes it a lot more complicated to query historical data.
Instead you should migrate the data to the new tables, you just need to copy `network_fees` to both `transaction_published` and `transaction_confirmed`.

Reply:
That's a good point, I didn't realize that the migration was in fact quite easy. I'll work on that!

Reply:
Done in https://github.com/ACINQ/eclair/pull/1976/commits/efba393128d9dd28c36440cc6f6036c8967464cd